lib/src/main/java/com/thenewboston/api/common/GetDataSource.kt

```diff
@@ -75,7 +75,11 @@ class GetDataSource @Inject constructor(private val networkClient: NetworkClient
         val urlSuffix = "$validatorsEndpoint/$nodeIdentifier"
         val response = networkClient.defaultClient.get<Validator>(urlSuffix)
 
-        return Outcome.Success(response)
+        return when {
+            response.nodeIdentifier.length != 64 ->
```

Comment:
I don't understand this condition. Why are we checking if the length is not 64?

Reply:
To theck if the nice if is valid

Reply:
The backend would ensure that's it is valid or not. No need to add this on our side.